examples/deployment/xcode_cloud/ci_post_clone.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# The default execution directory of this script is the ci_scripts directory.
+cd $CI_WORKSPACE # change working directory to the root of your cloned repo.
+
+# Install Flutter using git.
+git clone https://github.com/flutter/flutter.git -b stable $HOME/flutter
+export PATH="$PATH:$HOME/flutter/bin"
+
+# Install Flutter artifacts for iOS (--ios), or macOS (--macos) platforms.
+flutter precache --ios
+
+# Install Flutter dependencies.
+flutter pub get
+
+# Install CocoaPods using Homebrew.
+HOMEBREW_NO_AUTO_UPDATE=1 # disable homebrew's automatic updates.
+brew install cocoapods
+
+# Install CocoaPods dependencies.
+cd ios && pod install # run `pod install` in the `ios` directory.
+
+exit 0
```

Comment:
nit: please add trailing new line at end of file. Thanks!

Reply:
@domesticmouse Okay, I just added it